introduction.md

```diff
@@ -1,3 +1,10 @@
 Hi!  My name is Shaylee Hansen.  I'm taking this course in preparation for the Python/Django Web Development bootcamp that I'm starting at the end of September. 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?